internal/planner/plansdk/analyzer.go

```diff
@@ -57,7 +58,7 @@ func (a *Analyzer) GlobFiles(patterns ...string) []string {
 	return results
 }
 
-func (a *Analyzer) FileExists(relPath string) bool {
+func (a *Analyzer) FileExists(relPath string) bool { // TODO: remove it?
```

Comment:
yeah, lets replace it in callers with `fileutil.Exists(a.AbsPath(relPath))` ?

Reply:
I mean deleting this method because it's not used anywhere.